src/endian.h

```diff
@@ -0,0 +1,155 @@
+/* Copyright (C) Internet Systems Consortium, Inc. ("ISC") - see COPYING */
+
+#pragma once
```

Comment:
I would rather use the usual ifdef based guards here

Reply:
`#pragma once` is well supported even in ancient compilers, but ok

Reply:
Also with `#pragma once` you don't have to come up with fancy names like `_P11PROV_ENDIAN_H`.